content/en/docs/intro/using_helm.md

```diff
@@ -533,4 +533,4 @@ covered useful utility commands like `helm status`, `helm get`, and `helm repo`.
 For more information on these commands, take a look at Helm's built-in help:
 `helm help`.
 
-In the next chapter, we look at the process of developing charts.
+In the [next chapter](/docs/howto/charts_tips_and_tricks/), we look at the process of developing charts.
```

Comment:
Can you please setup your relative links like the example at https://raw.githubusercontent.com/helm/helm-www/master/content/en/docs/chart_template_guide/wrapping_up.md. This will enable them to work as a repo of markdown as well as a site. Notice, the relative is not from the site URL but from document to document in the directory.

Reply:
Done @mattfarina. Note, I had used the `Chart Development Guide` link on the page as an example, which used site relative links. I went ahead and updated that too.